internal/input/input_js.go

```diff
@@ -49,6 +49,11 @@ func (i *Input) ClearRuneBuffer() {
 	i.runeBuffer = nil
 }
 
+func (i *Input) ResetScrollValues() {
```

Comment:
I think `ResetScrollValues` is not needed in `input_js.go`

Reply:
True, I'll add a TODO comment in `ui_js.go`